doc/en/user/source/community/gsr/functionality.rst

```diff
@@ -0,0 +1,104 @@
+Functionality
+=====================
+ 
+Formats
+---------------------------
+
+While the ArcGis implementation of GeoServices supports multiple file formats, all responses from the GeoServer plugin use JSON (for example, browser-friendly HTML is not provided.)  The ``?f=json`` parameter specifying JSON is still required.
```

Comment:
You will need to use:

```
While the ArcGIS® ...
```

I hate trademarks

Reply:
fixed